### ISSUE TITLE
Makr static receiving addresses an option to choose

### ISSUE THREAD

**Issue by Phyloiota** [iotaledger/firefly#725]
## Description

A lot of users would be happy to have a static - never changing receiving address, I suggest that addresses stay generally changing, but that an account can have a option (switch or checkbox) that say "make this a static / never changing receiving address). With a popup that explains that it could be an issue regarding privacy if a receiving address can be linked to a person, but it the users agrees to take that small risk he can have the address permanently.

## Motivation

Address handling can be difficult. People would like to have a static address for one account where they could oversee incoming payments (online payments, etc etc) and which will be the same if they want to share it from time to time - so they could also see in the explorer all history for that address.
Other option would be to always generate new addresses by default, but have a box in every account with one static address fixed written into it so the user can decide ... if he wants to share the static address he copies this one, if he wants a new one he copies the fresh generated

## Requirements

some UI implementations like a switch or a fixed address field that keeps one address

## Open questions (optional)

If this inherets a serious security risk than dont do it, or if this is easier and safer to handle in an account based transaction and address archieve lets do it there, but the feedback so far from community suggests that tit would be a welcomed feature

## Are you planning to do it yourself in a pull request?

no

**Comment by cvarley100** [comment]
@Phyloiota can the user not go into the address history (once added) and copy it from there?

**Comment by Phyloiota** [comment]
Yes thats also possible and would be fine i think, just would involve a few more steps... and maybe leads to confusion. If you have a lot of generated addresses, how do you remember which one you used the last time as "static address"? So i thought it would be nicer to have it in a more prominent place shown and ready to copy so that users dont need to search around for it...And it would be directly obvious for new users that it is Possible to use this address again. You see - the solution now confuses people, more simple solutions for the "lazy layman" will be a welcomed thing ...   :)
